### PR TITLE
Tweak description of bb-summary-actionbar-cancel property adn tweak c…

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
…ancel buttons in demo.

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
We're going to need to add the implementation of the close method too

**Comment by Blackbaud-BobbyEarl** [comment]
LGTM.  We'll need to update the JS side of the demo to include the implementation of the `close` method as well.

**Comment by blackbaud-johnly** [comment]
Thanks @Blackbaud-BobbyEarl. I updated the JS file.

File: js/sky/src/summaryactionbar/docs/demo.js [hunk 1]
```diff
@@ -3,14 +3,14 @@
 (function () {
     'use strict';
 
-    function SummaryActionbarTestController() {
+    function SummaryActionbarTestController($uibModalInstance) {
         var ctrl = this;
 
-        function alertEmit(message) {
```
Comment (Blackbaud-BobbyEarl) [inline js/sky/src/summaryactionbar/docs/demo.js]: This is my fault so sorry for the confusion, I didn't think about this file already having an implementation of the `SummaryActionbarTestController` method.  The stuff that was already there will need to remain but adding the new stuff.

Comment (blackbaud-johnly) [reply js/sky/src/summaryactionbar/docs/demo.js]: I added the alertEmit function back in. Hopefully this is correct now.

**Review by Blackbaud-BobbyEarl — APPROVED** [review]

**Pull request merged**